test/integration/common/test_console_output.py

```diff
@@ -0,0 +1,64 @@
+from genty import genty, genty_dataset
+import os
+from tempfile import mkstemp
+from unittest import TestCase
+
+from app.common.console_output import ConsoleOutput
+
+
+@genty
+class TestConsoleOutput(TestCase):
```

Comment:
Should we create an (empty for now) `BaseIntegrationTestCase` class? I can see us definitely using that as we create more integration tests.